libcaliptra/inc/caliptra_types.h

```diff
@@ -347,6 +351,10 @@ struct dpe_destroy_context_cmd {
     uint32_t flags;
```

Comment:
I believe this should be removed.
Reference: https://github.com/TrustedComputingGroup/Server-Internal/blob/main/dpe-irot-profile/main.tcg#L584

Reply:
I'll ping you on this one. Unfortunately I can't open that link. I do see this still in the latest version of the DPE IROT profile I can find (v0.10) though.

Reply:
After talking with Ehsan, this field was not part of the actual implementation and is being removed from the spec